From 99f78609e2b81169e1ed4aa9ffa31aeadeb4ca52 Mon Sep 17 00:00:00 2001
From: Thorsten Behrens <tbehrens@novell.com>
Date: Wed, 23 Feb 2011 07:44:17 +0100
Subject: [PATCH] Temp fix for slideshow media playback bug.

Fixes deb#612940 and n#651250 by disabling the impress media
embedding patches. Those don't pass on proper temporary file system
urls to avmedia, during slideshow.
---
 patches/dev300/apply |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/patches/dev300/apply b/patches/dev300/apply
index c60565e..4df4ea2 100644
--- libreoffice-3.3.1/libreoffice-build/patches/dev300/apply
+++ libreoffice-3.3.1/libreoffice-build/patches/dev300/apply
@@ -2279,8 +2279,8 @@ transogl-crossplatform.diff, ericb
 [ Features ]
 # embed generic media files inside odf docs, plus various related
 # sound fixes from rodo
-impressmedia-features.diff, i#83753, n#515553, thorsten
-slideshow-sound.diff, n#515553, rodo
+#impressmedia-features.diff, i#83753, n#515553, thorsten
+#slideshow-sound.diff, n#515553, rodo

 [ Fixes ]
 vcl-gtk-process-events-on-exit.diff, rodo
--
1.6.4.2
